hubblestack/files/hubblestack_nova/win_reg.py

```diff
@@ -86,11 +88,13 @@ def audit(data_list, tags, debug=False, **kwargs):
                                 tag_data['value_found'] = current
                                 ret['Success'].append(tag_data)
                             else:
+                                tag_data['failure_reason'] = "Value of registry key '{0}:{1}:{2}' is set to value '{3}'. It should be set to '{4}({5})'.".format(reg_dict['hive'], reg_dict['key'], reg_dict['value'], current, match_output, tag_data['value_type'])
```

Comment:
Can you shorten these very long lines? I don't necessarily want to enforce the pep8 80 column rule, but I do like to aim for 90-ish columns with a hard cap at 120 columns.

For example, I would probably format this line like this:

```
                                tag_data['failure_reason'] = "Value of registry key '{0}:{1}:{2}' should be set to " \
                                                             "'{4}({5})'. It is set to some other value for one or " \
                                                             "more SID(s)".format(reg_dict['hive'],
                                                                                  reg_dict['key'],
                                                                                  reg_dict['value'],
                                                                                  current,
                                                                                  match_output,
                                                                                  tag_data['value_type'])
```